bin/developers.js

```diff
@@ -0,0 +1,9 @@
+const initDistribution = require('../src/commands/developers/init-distribution')
+const deploy = require('../src/commands/developers/deploy')
+const register = require('../src/commands/developers/register')
+
+module.exports = function processDevelopersCommands(program) {
```

Comment:
I'd rename `process` to `register`, but it's a detail

Reply:
agree!